feature/gribi/ate_tests/ipv4_entry_test/ipv4_entry_test.go

```diff
@@ -289,6 +301,28 @@ func TestIPv4Entry(t *testing.T) {
 
 			for _, tc := range cases {
 				t.Run(tc.desc, func(t *testing.T) {
+					//Creating a Static ARP entry for staticDstMAC
+					if deviations.GRIBIIPv4EntryMACOverride(ondatra.DUT(t, "dut")) && tc.desc == "Multiple next-hops with MAC override" {
```

Comment:
Instead of using string comparison, which is hard to maintain, can you create a new bool field in the cast struct? With that the testcase of "Multiple next-hops with MAC override" will be:

		{
			desc: "Multiple next-hops with MAC override",
			entries: []fluent.GRIBIEntry{ ... },
			wantGoodFlows: []*ondatra.Flow{ ... },
			wantOperationResults: []*client.OpResult{...},
                         gribiIPv4EntryMACOverride:  deviations.GRIBIIPv4EntryMACOverride(ondatra.DUT(t, "dut")),
		},

Reply:
updated testcase as per this.